### PR TITLE
reference/tools: update dm monitor

### PR DESCRIPTION
**Pull request by anotherrachel**
### What is changed, added or deleted? 
This PR updates DM monitor description, according to grafana.
- added alert severity level
- removed unused `Dumper` part
- renamed `dumper` to `dump unit`, renamed `loader` to `load unit`

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1870

### Which version does your change affect? 
dev, v2.1, v3.0

### CODE REVIEW

File: dev/reference/tools/data-migration/monitor.md [hunk 1]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| load process exits with error | Loader encounters an error within the DM-worker and exits. | Immediate alerts |
-| table count | The total number of tables in the full data imported by Loader | N/A |
-| data file count | The total number of data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| latency of execute transaction | The duration that Loader executes a transaction (in seconds) | N/A |
-| latency of query | The duration that Loader executes a query (in seconds) | N/A |
+| Metric name | Description | Alert | Level |
+|:----|:------------|:----|:----|
+| load progress | The data import process percentage of `load unit`. The value range is 0% ~ 100%. | N/A | N/A |
+| data file size | The total size of the data files in the full data imported by `load unit` (including the `INSERT INTO` statement) | N/A | N/A |
+| dump process exits with error | `dump unit` encounters an error within the DM-worker and exits. | Immediate alerts | critical |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| dump process exits with error | The dump unit encounters an error within the DM-worker and exits. | Immediate alerts | critical |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 2]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+| load process exits with error | `load unit` encounters an error within the DM-worker and exits. | Immediate alerts | critical |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| load process exits with error | The load unit encounters an error within the DM-worker and exits. | Immediate alerts | critical |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 3]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+| table count | The total number of tables in the full data imported by Loader | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| table count | The total number of tables in the full data imported by the load unit | N/A | N/A |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 4]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+| data file count | The total number of data files in the full data imported by `load unit` (including the `INSERT INTO` statement) | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| data file count | The total number of data files (includes the `INSERT INTO` statement) in the full data imported by the load unit | N/A | N/A |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 5]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+| latency of execute transaction | The duration that `load unit` executes a transaction (in seconds) | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| latency of execute transaction | The latency of executing a transaction by the load unit (in seconds) | N/A | N/A |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 6]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+| latency of query | The duration that `load unit` executes a query (in seconds) | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| latency of query | The latency of executing a query by the load unit (in seconds) | N/A | N/A |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 7]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| binlog event qps | The number of binlog events received per unit of time (this number does not include the events that need to be skipped) | N/A |
-| skipped binlog event qps | The number of binlog events received per unit of time that need to be skipped | N/A |
-| cost of binlog event transform | The time it takes `syncer` to parse and transform the binlog into SQL statements (in seconds) | N/A |
-| total sqls jobs | The number of newly added jobs per unit of time | N/A |
-| finished sqls jobs | The number of finished jobs per unit of time | N/A |
-| execution latency | The time it takes `syncer` to execute the transaction to the downstream (in seconds) | N/A |
-| unsynced tables | The number of tables that have not received the shard DDL statement in the current subtask | N/A |
-| shard lock resolving | Whether the current subtask is waiting for the shard DDL lock to be resolved; a value greater than 0 indicates that it is waiting for the shard DDL lock to be resolved | N/A |
+| Metric name | Description | Alert | Level |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| Metric name | Description | Alert | Severity level |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 8]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| fail to read binlog from master | The number of errors encountered when relay log reads the binlog from the upstream MySQL | Immediate alerts |
-| fail to write relay log | The number of errors encountered when the relay log writes the binlog to disks | Immediate alerts |
-| binlog file index | The largest index number of relay log files. For example, "value = 1" indicates "relay-log.000001" | N/A |
-| binlog file gap between master and relay | The number of binlog files by which the `relay` processing unit is behind the upstream master | An alert occurs when the number of binlog files by which the `relay` processing unit is behind the upstream master exceeds one (>1) and the condition lasts over 10 minutes |
-| binlog pos | The write offset of the latest relay log file  | N/A |
-| read binlog duration | The duration that the relay log reads the binlog from the upstream MySQL (in seconds) |  N/A |
-| write relay log duration | The duration that the relay log writes the binlog into the disk each time (in seconds) | N/A |
-| binlog size | The size of a single binlog event that the relay log writes into the disks | N/A |
+| Metric name | Description | Alert | Level |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| Metric name | Description | Alert | Severity level |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 9]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
 
 ### Task
 
-| Metric name | Description | Alert |
-|:----|:------------|:----|
-| task state | The state of subtasks for replication | An alert occurs when the subtask has been paused for more than 10 minutes |
-| load progress | The percentage of loading completion of loader. The value is between 0% and 100%  | N/A |
-| binlog file gap between master and syncer | The number of binlog files by which binlog replication is behind the upstream master  | N/A |
-| shard lock resolving | Whether the current subtask is waiting for sharding DDL replication. A value greater than 0 means that the current subtask is waiting for sharding DDL replication | N/A |
+| Metric name | Description | Alert | Level |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| Metric name | Description | Alert | Severity level |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 10]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
+|:----|:------------|:----|:----|
+| task state | The state of subtasks for replication | An alert occurs when the subtask has been paused for more than 10 minutes | critical |
+| load progress | The percentage of loading completion of `load unit`. The value is between 0% and 100%  | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| load progress | The percentage of the completed loading process of the load unit. The value range is 0%~100%  | N/A | N/A |
```

File: dev/reference/tools/data-migration/monitor.md [hunk 11]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| data file size | The total size of the data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| load process exits with error | Loader encounters an error within the DM-worker and exits. | Immediate alerts |
-| table count | The total number of tables in the full data imported by Loader | N/A |
-| data file count | The total number of data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| latency of execute transaction | The duration that Loader executes a transaction (in seconds) | N/A |
-| latency of query | The duration that Loader executes a query (in seconds) | N/A |
+| Metric name | Description | Alert | Severity level |
+|:----|:------------|:----|:----|
+| load progress | The percentage of the completed loading process of the load unit. The value range is 0%~100% | N/A | N/A |
+| data file size | The total size of the data files in the full data imported by the load unit (including the `INSERT INTO` statement) | N/A | N/A |
```
Comment (yikeke) [inline dev/reference/tools/data-migration/monitor.md]:
```suggestion
| data file size | The total size of the data files (includes the `INSERT INTO` statement) in the full data imported by the load unit | N/A | N/A |
```

File: v2.1/reference/tools/data-migration/monitor.md [hunk 12]
```diff
@@ -20,107 +20,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| data file size | The total size of the data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| load process exits with error | Loader encounters an error within the DM-worker and exits. | Immediate alerts |
-| table count | The total number of tables in the full data imported by Loader | N/A |
-| data file count | The total number of data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| latency of execute transaction | The duration that Loader executes a transaction (in seconds) | N/A |
-| latency of query | The duration that Loader executes a query (in seconds) | N/A |
+| Metric name | Description | Alert | Severity level |
+|:----|:------------|:----|:----|
+| load progress | The percentage of the completed loading process of the load unit. The value range is 0%~100%. | N/A | N/A |
+| data file size | The total size of the data files in the full data imported by the load unit (including the `INSERT INTO` statement) | N/A | N/A |
```
Comment (yikeke) [inline v2.1/reference/tools/data-migration/monitor.md]:
```suggestion
| data file size | The total size of the data files (includes the `INSERT INTO` statement) in the full data imported by the load unit | N/A | N/A |
```

File: v3.0/reference/tools/data-migration/monitor.md [hunk 13]
```diff
@@ -21,107 +21,100 @@ In the Grafana dashboard, the default name of DM is `DM-task`.
...
-| data file size | The total size of the data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| load process exits with error | Loader encounters an error within the DM-worker and exits. | Immediate alerts |
-| table count | The total number of tables in the full data imported by Loader | N/A |
-| data file count | The total number of data files in the full data imported by Loader (including the `INSERT INTO` statement) | N/A |
-| latency of execute transaction | The duration that Loader executes a transaction (in seconds) | N/A |
-| latency of query | The duration that Loader executes a query (in seconds) | N/A |
+| Metric name | Description | Alert | Severity level |
+|:----|:------------|:----|:----|
+| load progress | The percentage of the completed loading process of the load unit. The value range is 0%~100% | N/A | N/A |
+| data file size | The total size of the data files in the full data imported by the load unit (including the `INSERT INTO` statement) | N/A | N/A |
```
Comment (yikeke) [inline v3.0/reference/tools/data-migration/monitor.md]:
```suggestion
| data file size | The total size of the data files (includes the `INSERT INTO` statement) in the full data imported by the load unit | N/A | N/A |
```

**Comment by lilin90** [comment]
@anotherrachel Please address comments.

**Pull request merged**